conanfile.py

```diff
@@ -31,7 +30,7 @@ class UpCpp(ConanFile):
         "shared": False,
         "fPIC": False,
         "build_testing": False,
-        "build_unbundled": False,
+        "build_unbundled": True,
```

Comment:
Why is it important to change the default? Does that change the build instructions in the README file?